projects/angular/src/forms/styles/_textarea.clarity.scss

```
@@ -27,9 +28,14 @@
     border: tokens.$cds-alias-object-border-width-100 solid forms-variables.$clr-forms-border-color;
     background: forms-variables.$clr-forms-textarea-background-color;
     color: forms-variables.$clr-forms-text-color;
-    border-radius: forms-variables.$clr-forms-textarea-border-radius;
-    padding: tokens.$cds-global-space-7;
-    @include mixins.generate-typography-token('SECONDARY-13-RG-STD');
+    border-radius: density.$clr-base-border-radius-s;
```

Revert the change and make it inside _properties.file where the form token value is set.
```suggestion
    border-radius: forms-variables.$clr-forms-textarea-border-radius;
```